website/content/docs/concepts/policies.mdx

```diff
@@ -870,6 +870,36 @@ However, the _contents_ of policies are parsed in real-time whenever the token i
 As a result, if a policy is modified, the modified rules will be in force the
 next time a token, with that policy attached, is used to make a call to Vault.
 
+## UI Related
+
+The Vault UI may require different ACL capabilities compared to the CLI. When [enabled](/vault/docs/configuration/ui#activating-the-vault-ui) 
+and using the `default` policy the following paths are automatically included:
+
+ - [/sys/internal/ui/mounts](/vault/api-docs/system/internal-ui-mounts) 
+ - [/sys/internal/ui/resultant-acl](/vault/api-docs/system/internal-ui-resultant-acl) 
+
+These UI related API paths may not be changed or altered via any policy once the UI 
+is enabled.
+
+The [/sys/internal/ui/mounts](/vault/api-docs/system/internal-ui-mounts) API provide a 
+list of mounts as per what's typically visible when logging into Vault and observing the 
+personal cubbyhole for example. The listing of mounts are determined by [the `listing_visibility` parameter](/vault/api-docs/system/mounts#listing_visibility) 
+that may be configured during the initial enabling of the mount or otherwise 
+[tuned afterwards](/vault/api-docs/system/auth#tune-auth-method).
+
+The [/sys/internal/ui/resultant-acl](/vault/api-docs/system/internal-ui-resultant-acl) API 
+is required for appropriate adaptation to responses used by the UI and if it were
+inaccessible then you may observe related warning or error messages within the UI itself.
```

Comment:
```suggestion
Vault uses the [/sys/internal/ui/resultant-acl](/vault/api-docs/system/internal-ui-resultant-acl)
endpoint to repackage information used by the UI. If you do not have have
permission to call the `ui/resultant-acl` endpoint, you may receive warnings or
errors in the UI.
```

I'm not sure I actually understood what this block was trying to say, so may edits may not be entirely accurate.